rpc/wrtc_signaling_answerer.go

```diff
@@ -509,7 +509,11 @@ func (ans *webrtcSignalingAnswerer) answer(client webrtcpb.SignalingService_Answ
 		utils.PanicCapturingGoWithCallback(func() {
 			defer close(done)
 			if err := exchangeCandidates(); err != nil {
-				sendErr(err)
+				if filterEOF(err, ans.logger) == nil {
```

Comment:
there is no `haveInit` here because since the first thing the answerer does is receive a `webrtcpb.AnswerRequest_Init`, so `haveInit` will be  implicitly true if we ever make it this far

Reply:
also, this could just be simplified to `sendErr(filterEOF(err))` but I thought the log line indicating EOF during ICE candidate exchange would be useful